<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>430</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupAsli">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Image Asli</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_loadCitra">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Load Citra</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNamaFile">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>340</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupProcess">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>451</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Image Process</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>311</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_grayscale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Grayscale</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_edgeDet">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Edge Detection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_reset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_simpan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Simpan</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupCont">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="title">
      <string>Contrast</string>
     </property>
     <widget class="QPushButton" name="button_pluscont">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_mincont">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBright">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="title">
      <string>Brightness</string>
     </property>
     <widget class="QPushButton" name="button_plusbright">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_minbright">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="Button_gaussian">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Gaussian Filter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_contour">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Contour</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelHasil">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelUmur">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>340</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_Spasial">
    <property name="title">
     <string>Operasi Spasial</string>
    </property>
    <widget class="QMenu" name="menuKonvolusi">
     <property name="title">
      <string>Konvolusi</string>
     </property>
     <addaction name="actionKFiltering1"/>
     <addaction name="actionKFiltering2"/>
    </widget>
    <widget class="QMenu" name="menuMean_Filtering">
     <property name="title">
      <string>Mean Filtering</string>
     </property>
     <addaction name="actionMFiltering1"/>
     <addaction name="actionMFiltering2"/>
    </widget>
    <widget class="QMenu" name="menuSharpening">
     <property name="title">
      <string>Sharpening</string>
     </property>
     <addaction name="actionSharpening"/>
     <addaction name="actionLaplace"/>
    </widget>
    <addaction name="menuKonvolusi"/>
    <addaction name="menuMean_Filtering"/>
    <addaction name="actionGaussian_Filtering"/>
    <addaction name="menuSharpening"/>
    <addaction name="actionMedian_Filtering"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Geometri">
    <property name="title">
     <string>Operasi Geometri</string>
    </property>
    <widget class="QMenu" name="menuResize">
     <property name="title">
      <string>Resize</string>
     </property>
     <widget class="QMenu" name="menuZoomIn">
      <property name="title">
       <string>ZoomIn</string>
      </property>
      <addaction name="actionZoom_In_2x"/>
      <addaction name="actionZoom_In_3x"/>
      <addaction name="actionZoom_In_4x"/>
     </widget>
     <widget class="QMenu" name="menuZoomOut">
      <property name="title">
       <string>ZoomOut</string>
      </property>
      <addaction name="actionZoom_Out_1_2"/>
      <addaction name="actionZoom_Out_1_4"/>
      <addaction name="actionZoom_Out_3_5"/>
     </widget>
     <addaction name="menuZoomIn"/>
     <addaction name="menuZoomOut"/>
    </widget>
    <addaction name="actionTranslasi"/>
    <addaction name="menuResize"/>
    <addaction name="actionCrop"/>
   </widget>
   <widget class="QMenu" name="menuHistogram">
    <property name="title">
     <string>Histogram</string>
    </property>
    <addaction name="actionHGrayscale"/>
    <addaction name="actionHRGB"/>
    <addaction name="actionHEqualization"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperasi_Geometri"/>
   <addaction name="menuOperasi_Spasial"/>
   <addaction name="menuHistogram"/>
  </widget>
  <action name="actionOperasi_Pencerahan">
   <property name="text">
    <string>Operasi Pencerahan</string>
   </property>
  </action>
  <action name="actionSimple_Contrast">
   <property name="text">
    <string>Simple Contrast</string>
   </property>
  </action>
  <action name="actionContrast_Streching">
   <property name="text">
    <string>Contrast Streching</string>
   </property>
  </action>
  <action name="actionNegative_Image">
   <property name="text">
    <string>Negative Image</string>
   </property>
  </action>
  <action name="actionBiner_Image">
   <property name="text">
    <string>Biner Image</string>
   </property>
  </action>
  <action name="actionHistogram_Grayscale">
   <property name="text">
    <string>Histogram Grayscale</string>
   </property>
  </action>
  <action name="actionHistogram_RGB">
   <property name="text">
    <string>Histogram RGB</string>
   </property>
  </action>
  <action name="actionHistogram_Equalization">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="actionTranslasi">
   <property name="text">
    <string>Translasi</string>
   </property>
  </action>
  <action name="action45">
   <property name="text">
    <string>45</string>
   </property>
  </action>
  <action name="action45_Derajat">
   <property name="text">
    <string>45 Derajat</string>
   </property>
  </action>
  <action name="action_45_Derajat">
   <property name="text">
    <string>-45 Derajat</string>
   </property>
  </action>
  <action name="action45_Derajat_2">
   <property name="text">
    <string>45 Derajat</string>
   </property>
  </action>
  <action name="action_90_Derajat">
   <property name="text">
    <string>-90 Derajat</string>
   </property>
  </action>
  <action name="action90_Derajat">
   <property name="text">
    <string>90 Derajat</string>
   </property>
  </action>
  <action name="action180_Derajat">
   <property name="text">
    <string>180 Derajat</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionTambah_dan_Kurang">
   <property name="text">
    <string>Tambah dan Kurang</string>
   </property>
  </action>
  <action name="actionOperasi_AND">
   <property name="text">
    <string>Operasi AND</string>
   </property>
  </action>
  <action name="actionTranspose">
   <property name="text">
    <string>Transpose</string>
   </property>
  </action>
  <action name="action4x">
   <property name="text">
    <string>4x</string>
   </property>
  </action>
  <action name="action3x">
   <property name="text">
    <string>3x</string>
   </property>
  </action>
  <action name="action2x">
   <property name="text">
    <string>2x</string>
   </property>
  </action>
  <action name="actionZoom_In_2x">
   <property name="text">
    <string>Zoom In 2x</string>
   </property>
  </action>
  <action name="actionZoom_In_3x">
   <property name="text">
    <string>Zoom In 3x</string>
   </property>
  </action>
  <action name="actionZoom_In_4x">
   <property name="text">
    <string>Zoom in 4x</string>
   </property>
  </action>
  <action name="actionZoom_Out_3_4">
   <property name="text">
    <string>Zoom Out 3/4</string>
   </property>
  </action>
  <action name="actionZoom_Out_1_2">
   <property name="text">
    <string>Zoom Out 1/2</string>
   </property>
  </action>
  <action name="actionZoom_Out_1_4">
   <property name="text">
    <string>Zoom Out 1/4</string>
   </property>
  </action>
  <action name="actionZoom_Out_3_5">
   <property name="text">
    <string>Zoom Out 3/4</string>
   </property>
  </action>
  <action name="actionKali_dan_Bagi">
   <property name="text">
    <string>Kali dan Bagi</string>
   </property>
  </action>
  <action name="actionOperasi_OR">
   <property name="text">
    <string>Operasi OR</string>
   </property>
  </action>
  <action name="actionOperasi_XOR">
   <property name="text">
    <string>Operasi XOR</string>
   </property>
  </action>
  <action name="action1920x1080">
   <property name="text">
    <string>1920x1080</string>
   </property>
  </action>
  <action name="actionKFiltering1">
   <property name="text">
    <string>Kernel1</string>
   </property>
  </action>
  <action name="actionKFiltering2">
   <property name="text">
    <string>Kernel2</string>
   </property>
  </action>
  <action name="actionMFiltering1">
   <property name="text">
    <string>Kernel1</string>
   </property>
  </action>
  <action name="actionGaussian_Filtering">
   <property name="text">
    <string>Gaussian Filtering</string>
   </property>
  </action>
  <action name="actionMedian_Filtering">
   <property name="text">
    <string>Median Filtering</string>
   </property>
  </action>
  <action name="actionMax_Filtering">
   <property name="text">
    <string>Max Filtering</string>
   </property>
  </action>
  <action name="actionMin_Filtering">
   <property name="text">
    <string>Min Filtering</string>
   </property>
  </action>
  <action name="actionMFiltering2">
   <property name="text">
    <string>Kernel2</string>
   </property>
  </action>
  <action name="actionFiltering3">
   <property name="text">
    <string>Filtering3</string>
   </property>
  </action>
  <action name="actionFiltering4">
   <property name="text">
    <string>Filtering4</string>
   </property>
  </action>
  <action name="actionFiltering5">
   <property name="text">
    <string>Filtering5</string>
   </property>
  </action>
  <action name="actionFiltering6">
   <property name="text">
    <string>Filtering6</string>
   </property>
  </action>
  <action name="actionSharpening">
   <property name="text">
    <string>Sharpening3</string>
   </property>
  </action>
  <action name="actionSharpening2">
   <property name="text">
    <string>Sharpening2</string>
   </property>
  </action>
  <action name="actionSharpening3">
   <property name="text">
    <string>Sharpening3</string>
   </property>
  </action>
  <action name="actionSharpening4">
   <property name="text">
    <string>Sharpening4</string>
   </property>
  </action>
  <action name="actionSharpening5">
   <property name="text">
    <string>Sharpening5</string>
   </property>
  </action>
  <action name="actionSharpening6">
   <property name="text">
    <string>Sharpening6</string>
   </property>
  </action>
  <action name="actionLaplace">
   <property name="text">
    <string>Laplace</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionHGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionHRGB">
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="actionHEqualization">
   <property name="text">
    <string>Equalization</string>
   </property>
  </action>
  <action name="actionAND">
   <property name="text">
    <string>AND</string>
   </property>
  </action>
  <action name="actionOR">
   <property name="text">
    <string>OR</string>
   </property>
  </action>
  <action name="actionXOR">
   <property name="text">
    <string>XOR</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
